exploratory_analysis.ipynb

```diff
@@ -142,7 +142,7 @@
        "2    2008.0  "
       ]
      },
-     "execution_count": 11,
+     "execution_count": 2,
```

Comment:
The read_csv didn't work for me for some reason. I've attached a snapshot: 
![Screenshot from 2020-09-19 22-16-52](https://user-images.githubusercontent.com/19784733/93694901-35b8e600-fac6-11ea-815b-ce267c4a24b2.png)
I don't know what is going on, I should be able to access the file on lfs, right?

Reply:
Maybe I need to install LFS or something?

Reply:
> The read_csv didn't work for me for some reason. I've attached a snapshot:
 I think you need to click the file on the web site and click the download button on the right to download it to your local drive.